### PR TITLE
Renamed banner for fix issue number #14804.

### PR DESCRIPTION
**Pull request by KirilAngelov**

### CODE REVIEW

File: src/openrct2-ui/windows/TileInspector.cpp [hunk 1]
```diff
@@ -2094,26 +2094,6 @@ static void window_tile_inspector_paint(rct_window* w, rct_drawpixelinfo* dpi)
                 int16_t wallType = tileElement->AsWall()->GetEntryIndex();
                 DrawTextBasic(dpi, screenCoords, STR_TILE_INSPECTOR_WALL_TYPE, &wallType, { COLOUR_WHITE });
 
-                // Banner info
-                const auto wallEntry = tileElement->AsWall()->GetEntry();
-                if (wallEntry != nullptr && wallEntry->wall.flags & WALL_SCENERY_IS_BANNER)
```
Comment (duncanspumpkin) [inline src/openrct2-ui/windows/TileInspector.cpp:2099]: I meant remove this specific if check. Walls can have banners but the correct way to check if they exist or not is using `auto banner = tileElement->AsWall()->GetBanner();` and checking if the result is non null.

**Comment by duncanspumpkin** [comment]
There will be a few other locations in the codebase that you will need to replace with the new name.

File: src/openrct2/world/Scenery.h [hunk 2]
```diff
@@ -116,7 +116,7 @@ enum WALL_SCENERY_FLAGS
     WALL_SCENERY_HAS_PRIMARY_COLOUR = (1 << 0),   // 0x1
     WALL_SCENERY_HAS_GLASS = (1 << 1),            // 0x2
     WALL_SCENERY_CANT_BUILD_ON_SLOPE = (1 << 2),  // 0x4
-    WALL_SCENERY_IS_BANNER = (1 << 3),            // 0x8  // Probably indicates translucency
+    WALL_SCENERY_IS_DOUBLE_SIDED = (1 << 3),            // 0x8  // Probably indicates translucency
```
Comment (duncanspumpkin) [inline src/openrct2/world/Scenery.h:119]: You'll need to align the comment for the ci to pass. I would remove the "probably..." comment it is no longer relevant.

**Comment by duncanspumpkin** [comment]
You are still missing a few renames. Do a find and replace of `WALL_SCENERY_IS_BANNER` replacing all instances with `WALL_SCENERY_IS_DOUBLE_SIDED`

File: src/openrct2/object/WallObject.cpp [hunk 3]
```diff
@@ -120,6 +120,7 @@ void WallObject::ReadJson(IReadObjectContext* context, json_t& root)
                 { "hasTernaryColour",       WALL_SCENERY_HAS_TERNARY_COLOUR,    Json::FlagType::Normal },
                 { "hasGlass",               WALL_SCENERY_HAS_GLASS,             Json::FlagType::Normal },
                 { "isBanner",               WALL_SCENERY_IS_BANNER,             Json::FlagType::Normal },
```
Comment (duncanspumpkin) [inline src/openrct2/object/WallObject.cpp:122]: although counterintuitive this line needs the WALL_SCENERY_IS_BANNER replace with DOUBLE_SIDED version

Comment (KirilAngelov) [reply src/openrct2/object/WallObject.cpp]: Thank you for your help.

**Pull request merged**